frontend/src/components/ShootWorkers/GMachineType.vue

```diff
@@ -30,7 +30,7 @@ SPDX-License-Identifier: Apache-2.0
           <span v-if="item.raw.cpu">CPU: {{ item.raw.cpu }} | </span>
           <span v-if="item.raw.gpu">GPU: {{ item.raw.gpu }} | </span>
           <span v-if="item.raw.memory">Memory: {{ item.raw.memory }}</span>
-          <span v-if="item.raw.storage"> | Storage Class: {{ item.raw.storage.class }}</span>
+          <span v-if="item.raw.storage"> | Volume Type: {{ item.raw.storage.type }} | Class: {{ item.raw.storage.class }} | Default Size: {{ item.raw.storage.size }}</span>
```

Comment:
Do we want to keep this description?

Reply:
Peter and I agreed to keep it for now (as it was before)